frontend/projects/ui/src/app/pages/server-routes/server-backup/server-backup.page.html

```diff
@@ -1,14 +1,5 @@
-<!-- not backing up -->
-<ng-container *ngIf="!backingUp">
-  <backup-drives-header title="Create Backup"></backup-drives-header>
-  <ion-content class="ion-padding">
-    <backup-drives type="create" (onSelect)="presentModalPassword($event)"></backup-drives>
-  </ion-content>
-</ng-container>
-
 <!-- currently backing up -->
-<ng-container *ngIf="backingUp">
-
+<ng-container *ngIf="eosService.backingUp$ | async; else notBackingUp">
```

Comment:
This component gets rather huge. I would move backing up mode to a separate component to tidy it up.